File: MODUL 6/C/PRAK605-2310817220011-ReginaSilvaMaharatini.c
#include<stdio.h>
int main(){
        int mtrA[10][10], mtrB[10][10], mtrAxB[10][10];
        int a, b, c, d, jumlah=0;
        scanf("%d", &d);
        printf("Matriks A\n");
        for(a=0; a<d; a++){
          for(b=0; b<d; b++){
              scanf("%d", &mtrA[a][b]);}
              }  
        printf("Matriks B\n");
        for(a=0; a<d; a++){
          for(b=0; b<d; b++){
              scanf("%d", &mtrB[a][b]);}
              }
        for(a=0; a<d; a++){
          for(b=0; b<d; b++){
              for(c=0; c<d; c++){
                  jumlah = jumlah + mtrA[a][c] * mtrB[c][b];}      
                mtrAxB[a][b]= jumlah;
                jumlah = 0;}}
        printf("Matriks AXB\n");
        for (a=0; a<d; a++){
          for(b=0; b<d; b++){
                    printf("%d ", mtrAxB[a][b]);}
          printf("\n");}}
          

File: MODUL 6/Python/PRAK605-2310817220011-ReginaSilvaMaharatini.py
n = int(input())
mtrA = []
print("Matriks A")
for i in range (n):
    k = list(map(int, input().split()))
    mtrA.append(k)
mtrB = []
print("Matriks B")
for i in range (n):
    k = list(map(int, input().split()))
    mtrB.append(k)
mtrAxB = []
print("Matriks AXB")
for i in range (n):
    mtrAxB.append([])
    for j in range (n):
        x = 0
        for k in range (n):
            x = x + mtrA[i][k] * mtrB[k][j]
        mtrAxB[i].append(x)
for baris in mtrAxB:
    for k in baris:
        print(k, end=' ')
    print()